src/transformers/generation/utils.py

```diff
@@ -1234,6 +1234,17 @@ def generate(
                 "(https://huggingface.co/docs/transformers/main/en/main_classes/text_generation)"
             )
 
+        has_default_min_length = kwargs.get("min_length") is None and generation_config.min_length == 0
+        if has_default_max_length and generation_config.min_new_tokens is not None:
+            generation_config.min_length = generation_config.min_new_tokens + input_ids_seq_length
```

Comment:
We don't need to set `min_length` as we have a dedicated logits processor :)

These lines can be removed and, instead, while setting the logits processor list, we can add a `MinNewTokensLengthLogitsProcessor` as defined in #20892 (e.g. after [this line](https://github.com/huggingface/transformers/blob/305f41e4dea99d9fdc212287263b3748aacc63d3/src/transformers/generation/utils.py#L811))

Reply:
Sure, I will make a commit